crawler.rb

```diff
@@ -2,10 +2,16 @@
 
 require 'fileutils'
 
+mode = ARGV.shift || ""
+
+unless mode.empty?
+  mode = "-#{mode}"
```

Comment:
We have only `force` mode. All other values should be ignored.

Reply:
Done, anything that is != "force" considered as ""